style.css

```diff
@@ -77,17 +77,17 @@ td:hover {
   cursor: pointer;
   font-family: inherit;
 }
-.btn--start:hover,
-.btn--next:hover {
+#btn--start:hover,
```

Comment:
why did you change classes here, let's leave as it was before and use id's for the purpose of translation

Reply:
Ok I'will fix it